### PR TITLE
Update demo and documentation for new onRedeem parameters

### PR DESCRIPTION
**Pull request by victor-tremendous**

### CODE REVIEW

File: docs/documentation.md [hunk 1]
```diff
@@ -120,8 +120,8 @@ This approach requires more configuration, as rewards will have to be approved b
           onError: function(err) {
             console.log(err);
           },
-          onRedeem: function() {
-            console.log("Reward redeemed")
+          onRedeem: function(rewardId, orderId) {
+            console.log(`Reward redeemed: ${rewardId}. Order ID: ${orderId}`)
```
Comment (kurko) [inline docs/documentation.md:124]: Given this is within <script></script>, maybe we should use ES5 as examples?

File: docs/documentation.md [hunk 2]
```diff
@@ -120,8 +120,8 @@ This approach requires more configuration, as rewards will have to be approved b
           onError: function(err) {
             console.log(err);
           },
-          onRedeem: function() {
-            console.log("Reward redeemed")
+          onRedeem: function(rewardId, orderId) {
```
Comment (kurko) [inline docs/documentation.md:123]: I see the first block doesn't have the arguments. Just wondering if it's makes sense to add a note somewhere about it or not. I'm ambivalent I think, but wanted to surface the thought anyway

**Review by kurko — APPROVED** [review]

File: docs/documentation.md (hunk 1 above)
Comment (victor-tremendous) [reply docs/documentation.md:124]: Does string interpolation not work inside a `<script>` tag? 😮

Comment (kurko) [reply docs/documentation.md:124]: It's an ES6 thing, which we write in the backend, but browsers don't support it (it's possible Chrome support it nowadays, need to confirm). We then build the JS transpiling from ES6 to ES5 before responding to the frontend

Comment (victor-tremendous) [reply docs/documentation.md:124]: Looks like [all major browsers support it](https://caniuse.com/template-literals) (97% of users).

**Pull request merged**